routes/web.py

```diff
@@ -1,7 +1,12 @@
+from masonite.helpers.routes import group
 from masonite.routes import Get
 
 
 ROUTES = [
     Get().route('/test', None).middleware('auth'),
     Get().domain('test').route('/test', None).middleware('auth'),
+    group('/example', [
+        Get().route('test/1', 'TestController@show'),
```

Comment:
this route should here should have forward slashes in the beginning. let's keep it in line with the rest of the routes

Reply:
Ok